README.md

```diff
@@ -14,6 +14,13 @@ in Bazel, and by the
 [`swift_library` rule](https://github.com/bazelbuild/rules_swift/blob/master/doc/rules.md#swift_library)
 available from [rules_swift](https://github.com/bazelbuild/rules_swift).
 
+The goal of rules_ios has always been to allow seamlessly transitioning a set of targets from being built
+in Xcode to being built in bazel, with minimal-to-no code changes.
+In that vein, it acts as glue between the underlying rules (rules_apple and rules_swift and the native objc rules),
```

Comment:
Also it'd be cool to document was the goal of a "framework packaging" packaging rule and what problems it is solving